lib/tic_tac_toe_api_web/views/page_view.ex

```diff
@@ -4,4 +4,15 @@ defmodule TicTacToeApiWeb.PageView do
   def render("index.json", %{board: board}) do
     %{spots: board.spots}
   end
+
+  def render("status.json", %{result: result}) do
+    case result do
+      {:play, next_player, current_player, _spots} ->
+        %{result: "play", current_player: current_player, next_player: next_player}
+      {:win, winner, _spots} ->
+        %{result: "wins", Winner: winner}
+      {:tie, _message, _spots} ->
+        %{result: "TIE"}
+    end
+  end
```

Comment:
Hmmm, I don't love how the JSON is changing. As a someone using your API I don't expect to get different JSON. I would always expect the same fields, nothing added, nothing taken away. Can we always return the same set of fields to the user?

Reply:
To build off what Andy's comment, I have often found that the need to change the structure of the JSON can be a code smell that the underlying structure needs adjusting.

In this case, I kind of see four different possible outcomes: x wins, o wins, draw, and in progress. Would refactoring this response to have one field with four values be a possible improvement?

Reply:
Also, if you decide to stick with two distinct fields, I would recommend using `winner: nil` instead of `winner: "none"`